### PR TITLE
Support `torch.bitwise_invert` alias 

### LINKED ISSUE #71686: Support the `bitwise_invert` alias to 'bitwise_not`
### 🚀 The feature, motivation and pitch

Python Array API proposes the use of `bitwise_invert` among other Element-wise Functions. Currently, PyTorch supports the same functionality with `torch.bitwise_not`. As listed in #58742, to make it compliant with the Array API this issue proposes to add the `bitwise_invert` alias to `bitwise_not`

### Additional context

- [Python Array API bitwise_invert](https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html?highlight=bitwise_invert#bitwise-invert-x)

### PR DESCRIPTION
**Pull request by Dbhasin1**
Fixes #71686 

This PR adds the `bitwise_invert` alias to `bitwise_not` and also to its In-place version. 

Thanks, @kshitij12345 for guiding me on my first PR!

### CODE REVIEW

**Comment by kshitij12345** [comment]
@Dbhasin1 took a cursory look. Looks really good! Just rebase with the latest master once so that the CI can run.

Looks like symbols in `aten_interned_strings` are generated automatically now (reference https://github.com/pytorch/pytorch/issues/69407). So `aten_interned_strings.h` and `interned_strings.h` might not need any changes now.

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -664,6 +664,20 @@ def add_docstr_all(method, docstr):
...
+bitwise_invert() -> Tensor
+
+See :func:`torch.bitwise_invert`
+""")
+
+add_docstr_all('bitwise_invert_',
+               r"""
+bitwise_invert_() -> Tensor
+
+In-place version of :meth:`~Tensor.bitwise_invert`
```
Comment (kshitij12345) [inline torch/_tensor_docs.py:678]: Should we also mention that this is an alias of `bitwise_not`. 

Cause from here user will go to `bitwise_invert` method and from there to `torch.bitwise_invert` to realize that it is an alias.

File: torch/_tensor_docs.py [hunk 2]
```diff
@@ -664,6 +664,20 @@ def add_docstr_all(method, docstr):
 In-place version of :meth:`~Tensor.bitwise_not`
 """)
 
+add_docstr_all('bitwise_invert',
+               r"""
+bitwise_invert() -> Tensor
+
+See :func:`torch.bitwise_invert`
```
Comment (kshitij12345) [inline torch/_tensor_docs.py:671]: Should we also mention that this is an alias for `bitwise_not`.

**Review by kshitij12345 — APPROVED** [review]
LGTM! Thanks @Dbhasin1

File: aten/src/ATen/native/native_functions.yaml [hunk 3]
```diff
@@ -944,6 +944,15 @@
   dispatch:
     CPU, CUDA: bitwise_not_out
 
+# bitwise_invert, alias for bitwise_not
```
Comment (mruberry) [inline aten/src/ATen/native/native_functions.yaml:947]: Nice comment

File: aten/src/ATen/native/native_functions.yaml [hunk 4]
```diff
@@ -944,6 +944,15 @@
...
+- func: bitwise_invert(Tensor self) -> Tensor
```
Comment (mruberry) [inline aten/src/ATen/native/native_functions.yaml:948]: This signatures look good but I think they can also use the 

```
device_check: NoCheck
```

annotation

File: torch/_tensor_docs.py (hunk 2 above)
Comment (mruberry) [reply torch/_tensor_docs.py:671]: For whatever reason we typically have the method point to the function variant and then the function variant says it's an alias.

File: torch/_tensor_docs.py (hunk 1 above)
Comment (mruberry) [reply torch/_tensor_docs.py:678]: Yeah but (see above) that's just how it is for all aliases at the moment. We could think about updating all of them in a separate PR

**Review by mruberry — APPROVED** [review]
Nice work @Dbhasin1! Would you address the one small comment I made about the schemas and then ping me to get this merged?

**Comment by kshitij12345** [comment]
Gentle ping @mruberry

**Pull request closed without merging**